src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/src/DriveTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
You should have a negative test, to make sure when you try to get an invalid drive it returns 404 like expected.  Also, if there are drives that exist but should never be retrieved you should add a test for that too!

Reply:
I have added some negative tests to test invalid value for Drive, StorageSubSystem and Volume.